src/Command/LinkDependenciesCommand.php

```diff
@@ -73,7 +72,8 @@ protected function configure(): void
         $this
             ->setName('dependencies:link')
             ->setDescription(sprintf('Outputs a %s file that contains data about related Pull Requests', self::DEPENDENCIES_FILE))
-            ->addArgument('token', InputArgument::OPTIONAL, 'GitHub OAuth token')
+            ->addOption('token', 't', InputOption::VALUE_OPTIONAL, 'GitHub OAuth token')
```

Comment:
```suggestion
            ->addOption('token', 't', InputOption::VALUE_REQUIRED, 'GitHub OAuth token')
```

What's the point of providing only `-t` in cmd?

Reply:
> What's the point of providing only `-t` in cmd?

@adamwojs I'm not sure, token was there before, I just moved it to the options. We'd need to ask the author of this.

Reply:
Let's just say I'm not too attached to this shortcut, we can remove it 😄